docs/changelogs/v0.35.md

```diff
@@ -77,6 +77,8 @@ The release updated configuration [profiles](https://github.com/ipfs/kubo/blob/m
 - Updated Profile: `test-cid-v1` now includes current defaults as explicit `Import.UnixFSFileMaxLinks=174`, `Import.UnixFSDirectoryMaxLinks=0`, `Import.UnixFSHAMTDirectoryMaxFanout=256` and `Import.UnixFSHAMTDirectorySizeThreshold=256KiB`
 - New Profile: `test-cid-v1-wide` adopts experimental directory DAG-shaping defaults, increasing the maximum file DAG width from 174 to 1024, HAMT fanout from 256 to 1024, and raising the HAMT directory sharding threshold from 256KiB to 1MiB, aligning with 1MiB file chunks.
   - Feedback: Try it out and share your thoughts at [discuss.ipfs.tech/t/should-we-profile-cids](https://discuss.ipfs.tech/t/should-we-profile-cids/18507) or [ipfs/specs#499](https://github.com/ipfs/specs/pull/499).
+- When ipfs is initialized, the datastore metrics wrapper is not configured by default as it previously was. To enable datastore metrics during initialization, specifying the appropriate `--profile` option:
```

Comment:
nit: changelog should state which metrics are removed

```suggestion
- When ipfs is initialized, the datastore metrics wrapper is not configured by default as it previously was. This means metrics `TODO`, `BAR` are no longer exposed by default. To enable datastore metrics during initialization, specifying the appropriate `--profile` option:
```

Reply:
Done